src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -286,6 +294,26 @@ public async Task VaultTimeoutAsync(bool promptOptions = true, int? newTimeout =
             }
         }
 
+        public async Task AppCenterReportingAsync()
+        {
+#if !FDROID && !DEBUG
+            var options = _appcenterActions.Select(s => s == _appCenterReportsDisplayValue ? $"✓ {s}" : s).ToArray();
+            var selection = await Page.DisplayActionSheet(AppResources.ReportAppLogsDescription, AppResources.Cancel, null, options);
+
+            if (selection == null || selection == AppResources.Cancel)
+            {
+                return;
+            }
+
+            var cleanSelection = selection.Replace("✓ ", string.Empty);
+            var enableAppcenter = cleanSelection == AppResources.Yes;
+            await Microsoft.AppCenter.Crashes.Crashes.SetEnabledAsync(enableAppcenter);
+            _appCenterReportsEnabled = await Microsoft.AppCenter.Crashes.Crashes.IsEnabledAsync();
+            _appCenterReportsDisplayValue = _appCenterReportsEnabled ? AppResources.Enabled : AppResources.Disabled;
```

Comment:
if the options array is created in situ using the `_appCenterReportsEnabled` flag, then we can delete this line and the `_appCenterReportsDisplayValue` field